src/lang/en.json

```diff
@@ -33,6 +33,21 @@
         "Cancel": "Cancel",
         "Mystify": "Mystify",
         "MystifyReplace": "Replace"
+      },
+      "resetsettings": {
+        "title": "Reset Module Settings",
+        "content": "Are you sure you want to reset all of the module settings to the current system's defaults? <strong>THIS CANNOT BE UNDONE!</strong>",
+        "confirm": "Reset Module Settings",
+        "cancel": "Cancel"
+      },
+      "nosystemfound": {
+        "title": "System Not Recognized",
+        "content": "This system is not currently fully supported. This means that the module does not know how to best handle this system's items, but can still operate in a limited fashion.<br><br>You can configure the module's settings yourself, but it requires a bit of technical know-how."
+      },
```

Comment:
I'm unsure about this. Where in the code is this used? Is it a notification, or something that appears when the settings are rendered?

Reply:
As above, since there is no generic module management mechanism, the message in question does not make much sense, it should appear when the current system is not present among the systems registered by the module. This is perhaps a stretch on my part because I can't stop people from downloading the zip file and modifying the module.json to install it on an unmanaged system, I don't think this check hurts, but let me know it.

Reply:
I think that if someone is manually modifying files in order to get the module working on their system, they are already aware of the risks.